ambari-server/src/main/java/org/apache/ambari/server/orm/entities/ServiceGroupEntity.java

```diff
@@ -80,6 +84,14 @@
   @OneToMany(mappedBy="serviceGroupDependency")
   private List<ServiceGroupDependencyEntity> dependencies;
 
+  @ElementCollection()
+  @CollectionTable(name = "servicegroup_mpacknames",
+                   joinColumns = {@JoinColumn(name = "service_group_id", referencedColumnName = "id"),
+                                  @JoinColumn(name = "service_group_cluster_id", referencedColumnName = "cluster_id")})
+  @Column(name = "mpack_name", unique = true, nullable = false)
```

Comment:
I don't think the `mpack_name` in itself should be unique.  This causes unit test failures with the current constant `"dummy"` mpack name for tests.  (eg. `AmbariManagementControllerTest`)

Reply:
Actually mpack_name should not be unique. It should allow different service group to have the same mpack_name. I will remove this contraint.